geospaas_harvesting/harvesters.py

```diff
@@ -218,3 +218,14 @@ def _create_crawlers(self):
 class NetCDFLocalHarvester(LOCALHarvester):
     """Harvester class for one-dimensional NetCDF file hosted locally"""
     ingester = ingesters.NetCDFIngester
+
+
+class CustomizedNetCDFLocalHarvester(LOCALHarvester):
+    """Harvester class for customized NetCDF file hosted locally"""
+    def _create_ingester(self):
+        parameters = {}
+        for parameter_name in ['max_fetcher_threads', 'max_db_threads',
+                               'longitude_attribute', 'latitude_attribute']:
+            if parameter_name in self.config:
+                parameters[parameter_name] = self.config[parameter_name]
+        return ingesters.NetCDFIngester(**parameters)
```

Comment:
Let's avoid code duplication:
- no need to create a new class, `longitude_attribute` and `longitude_attribute` are relevant for the `NetCDFLocalHarvester`.
- rather than copy-paste the code from `WebDirectoryHarvester`, just make the list of arguments a class attribute in `WebDirectoryHarvester` and override it in `NetCDFLocalHarvester`.

Reply:
Done